Sources/Plasma/FeatureLib/pfGameGUIMgr/pfGUIMenuItem.h

```diff
@@ -117,8 +119,8 @@ class pfGUIMenuItem : public pfGUIButtonMod
         void    PurgeDynaTextMapImage() override;
 
 
-        void        SetName( const wchar_t *name );
-        const wchar_t   *GetName() const { return fName; }
+        void        SetName(ST::string name);
+        const ST::string& GetName() const { return fName; }
```

Comment:
Why not return `ST::string` (a non-const value)?

Reply:
Good question! I think I didn't know the convention here and just blindly translated all `const wchar_t *` to `const ST::string&`. Apparently getters urually return `ST::string` by value - I'll adjust it.